django_stubs_ext/setup.py

```diff
@@ -0,0 +1,43 @@
+from distutils.core import setup
+
+from setuptools import find_packages
+
+with open("README.md") as f:
+    readme = f.read()
+
+dependencies = [
+    "django",
+    "django-stubs",
+]
+
+setup(
+    name="django-stubs-ext",
+    version="0.1.0",
+    description="Monkey-patching and extensions for django-stubs",
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    license="MIT",
+    url="https://github.com/typeddjango/django-stubs",
+    author="Maksim Kurnikov",
```

Comment:
You are the author! 🙂

Reply:
thanks, i appreciate that! figured i'd leave it since it's your project :p

Reply:
ok, let's stick to our roots and make @mkurnikov the one in charge 😆

Reply:
i'm too tired to read usernames right now haha

Reply:
I'd appreciate it if you leave me as the main author in all those places. But if it matters, we could add the `CONTRIBUTORS.md` file with proper acknowledgments, what do you think?

Reply:
@mkurnikov sure thing! 👍